src/main/scala/centaur/test/standard/StandardTestFormat.scala

```diff
@@ -28,11 +27,9 @@ object StandardTestFormat {
   }
 
   def fromString(testFormat: String): ErrorOr[StandardTestFormat] = {
-    testFormat.toLowerCase match {
-      case "workflowsuccess" => Valid(WorkflowSuccessTest)
-      case "workflowfailure" => Valid(WorkflowFailureTest)
-      case "submissionfailure" => Valid(SubmissionFailureTest)
-      case bad => invalidNel(s"No such test format: $bad")
-    }
+    val formats = List(WorkflowSuccessTest, WorkflowFailureTest)
+    formats collectFirst {
+      case format if format.name.equalsIgnoreCase(testFormat) => Valid(format)
```

Comment:
I don't think that this works w/o changing the spec for the .test files (and subsequently all of the .test files) - for instance hello.test has `testFormat: workflowsuccess` and that doesn't match any of your formats. 

One could argue that the format in the `.test` file should have the whole word (I don't really care, it just seemed like less typing) but I still say that the previous scheme was easier to visually parse considering how few cases there were